conf/rates/alcohol-duty-rates.json

```diff
@@ -4,528 +4,890 @@
         "rateType": "Core",
-        "alcoholRegime": [
-          "Cider"
+        "alcoholRegimes": [
+          {
+            "name":"Cider",
+            "abvRanges": [
+              {
+                "name": "Cider",
+                "minABV": 1.3,
+                "maxABV": 3.4
+              }
+            ]
+          }
         ],
-        "minABV": 1.3,
-        "maxABV": 3.4,
         "rate": 9.27
       },
       {
         "taxType": "313",
         "description": "Wine (including sparkling wine) from 1.3% to 3.4%",
         "rateType": "Core",
-        "alcoholRegime": [
-          "Wine"
+        "alcoholRegimes": [
+          {
+            "name":"Wine",
+            "abvRanges": [
+              {
+                "name": "Wine",
+                "minABV": 1.3,
+                "maxABV": 3.4
+              }
+            ]
+          }
         ],
-        "minABV": 1.3,
-        "maxABV": 3.4,
         "rate": 9.27
       },
       {
         "taxType": "314",
         "description": "Other fermented products like fruit ciders from 1.3% to 3.4%",
         "rateType": "Core",
-        "alcoholRegime": [
-          "OtherFermentedProduct"
+        "alcoholRegimes": [
+          {
+            "name":"OtherFermentedProduct",
+            "abvRanges": [
+              {
+                "name": "OtherFermentedProduct",
+                "minABV": 1.3,
+                "maxABV": 3.4
+              }
+            ]
+          }
         ],
-        "minABV": 1.3,
-        "maxABV": 3.4,
         "rate": 9.27
       },
       {
         "taxType": "315",
         "description": "Spirits or spirit based products from 1.3% to 3.4%",
         "rateType": "Core",
-        "alcoholRegime": [
-          "Spirits"
+        "alcoholRegimes": [
+          {
+            "name":"Spirits",
+            "abvRanges": [
+              {
+                "name": "Spirits",
+                "minABV": 1.3,
+                "maxABV": 3.4
+              }
+            ]
+          }
         ],
-        "minABV": 1.3,
-        "maxABV": 3.4,
         "rate": 9.27
       },
       {
         "taxType": "321",
         "description": "Beer from 3.5% to 8.4%",
         "rateType": "Core",
-        "alcoholRegime": [
-          "Beer"
+        "alcoholRegimes": [
+          {
+            "name": "Beer",
+            "abvRanges": [
+              {
+                "name": "Beer",
+                "minABV": 3.5,
+                "maxABV": 8.4
+              }
+            ]
+          }
         ],
-        "minABV": 3.5,
-        "maxABV": 8.4,
         "rate": 21.01
       },
       {
         "taxType": "322",
         "description": "Cider (but not sparkling cider) from 3.5% to 8.4 or Sparkling cider from 3.5% to 5.5%%",
         "rateType": "Core",
-        "alcoholRegime": [
-          "Cider"
-        ],
-        "minABV": 3.5,
-        "maxABV": 8.4,
+        "alcoholRegimes": [
```

Comment:
Can we change thestructure to:

`        //RateBand
        "taxTypeCode": "322",
        "description": "Cider (but not sparkling cider) from 3.5% to 8.4 or Sparkling cider from 3.5% to 5.5%%",
        "rateType": "Core",
        "rate": 9.67,
        "rangeDetails": [
          {
            // RangeDetailsByRegime
            "alcoholRegime": "Cider",
            "abvRanges": [
              {
                // AbvRange
                "alcoholType": "Cider",
                "minABV": 3.5,
                "maxABV": 8.4
              },
              {
                "alcoholType": "SparklingCider",
                "minABV": 3.5,
                "maxABV": 5.5
              }
            ]
          }
        ]
      },`